pkg/placement/raft/server.go

```diff
@@ -189,6 +179,30 @@ func (s *Server) StartRaft(config *raft.Config) error {
 	return err
 }
 
+func (s *Server) bootstrapConfig(peers []PeerInfo) (*raft.Configuration, error) {
+	hasState, err := raft.HasExistingState(s.logStore, s.stableStore, s.snapStore)
+	if err != nil {
+		return nil, err
+	}
+
+	if !hasState {
+		raftConfig := &raft.Configuration{
+			Servers: make([]raft.Server, len(peers)),
+		}
+
+		for i, p := range peers {
+			raftConfig.Servers[i] = raft.Server{
+				ID:      raft.ServerID(p.ID),
+				Address: raft.ServerAddress(p.Address),
+			}
+		}
+
+		return raftConfig, nil
+	}
+
+	return nil, nil
```

Comment:
Please add a comment to explain why we return `nil` when already having state?

Reply:
added comment